### DISCUSSION TITLE
Metronome doesn't start immediately upon unpausing in editor timing panel.

### DISCUSSION THREAD

**Discussion by Nyanaro** [ppy/osu#25266] [Ideas] [answered]
The most common method of checking if an offset or BPM is correct in a short timing window is repeating a section of music and then listening if the metronome and song are in sync. Currently in the Lazer editor the metronome does not start ticking until a certain amount of time has passed, making this method impossible.

While this isn't a massive issue for single-bpm songs, for variable BPM it makes timing very frustrating. I would still rather time my maps in stable over Lazer because of this. Video attached showcasing this issue.

My suggestion is to have the metronome function similarly to stable, where it plays a tick every time it passes a white tick, without the delay (or at least having a toggle for this function).

https://github.com/ppy/osu/assets/22597387/112e756c-7983-46b3-96df-65faedc6a635

**Comment by peppy** [comment]
This was intentional to make the metronome behave correctly (as to its physical representation), but I can understand the scenario you mention, for users that are used to timing via old methodology.

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/pull/25316
